modules/authentication/proc-enabling-authentication-with-rhbk.adoc

```diff
@@ -141,14 +105,19 @@ auth:
       production:
         scope: ${AUTH_OIDC_SCOPE}
 ----
---
 
-`signIn.resolvers`::
---
-Declarative resolvers to override the default resolver: `emailLocalPartMatchingUserEntityName`.
+`signIn`::
+`resolvers`:::
+Enter the resolver list to override the default resolver: `emailLocalPartMatchingUserEntityName`.
 The authentication provider tries each sign-in resolver until it succeeds, and fails if none succeed.
```

Comment:
After successful authentication, the user signing in must be resolved to an existing user in the {product-short} catalog. To best match users securely for your use case, consider configuring a specific resolver. Enter the resolver list to override the default resolver: `emailLocalPartMatchingUserEntityName`.
+
The authentication provider tries each sign-in resolver in order until it succeeds, and fails if none succeed.
+
WARNING: In production mode, only configure one resolver to ensure users are securely matched.

Reply:
```suggestion
After successful authentication, the user signing in must be resolved to an existing user in the {product-short} catalog. To best match users securely for your use case, consider configuring a specific resolver. Enter the resolver list to override the default resolver: `emailLocalPartMatchingUserEntityName`.
+
The authentication provider tries each sign-in resolver in order until it succeeds, and fails if none succeed.
+
WARNING: In production mode, only configure one resolver to ensure users are securely matched. 
```